dr-cabal.cabal

```diff
@@ -81,8 +81,11 @@ library
   exposed-modules:
     DrCabal
       DrCabal.Cli
+      DrCabal.Format
```

Comment:
This could be `DrCabal.Profile.Format` as we most likely won't have any reusable formatting outside profiling